### DISCUSSION TITLE
Any ETA for the recursion fix to appear as a new NuGet version?

### DISCUSSION THREAD

**Discussion by UweKeim** [cefsharp/CefSharp#3916] [General]
We've run into [the recursion issue that was recently fixed](https://github.com/cefsharp/CefSharp/commit/51b2c74801e1c24031d6624659be5a79b3235ddf).

Since we want to use this in production ASAP (and currently can't because I updated the NuGet package too early and thus now have the version with the issue in my application), my question is:

Is there any timeline estimation when this fix will be available as a new NuGet package?

**Comment by amaitland** [comment]
Original plan was yesterday, unfortunately I've had some water ingress into our garage so I've had to deal with that as a priority.

New packages should be available within the hour.

Workaround incase you need a fix sooner is pretty straight forward

```c#
//Replace
chromiumWebBrowser.ExecuteScriptAsync(method, args);
//With
chromiumWebBrowser.GetBrowser().ExecuteScriptAsync(method, args);
```

**Reply by UweKeim** [reply]
Awesome. Thank you, Alex! Hope your house and garage (and of course your family) is safe!

**Reply by amaitland** [reply]
> Hope your house and garage (and of course your family) is safe!

Thanks, everyone is fine. It's an old house, so it's just one of those things that happens when there is a lot of water.

**Reply by UweKeim** [reply]
Hooray! It arrived on NuGet. Thanks again for your awesome, fast help 😊.
